main.cpp

```diff
@@ -51,85 +51,96 @@ arm.flex();                             // 3)
 arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
-//  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  1)
+//  Noun: washingMachine
+//  action 1: the washing machine washes clothes
+washingMachine.washClothes();
+//  action 2: the washing machine rinses clothes
+washingMachine.rinseClothes();
+//  action 3: the washing machine spins
+washingMachine.spin();
+
+//  2)
+//  Noun: dog
+//  action 1: the dog barks
+dog.bark();
+//  action 2: the dog fetches a ball
+dog.fetchBall();
+//  action 3: the dog wags its tail
+dog.wagTail();
+
+//  3)
+//  Noun: refrigerator
+//  action 1: the refrigerator goes for a jog
+refrigerator.goForJog();
+//  action 2: the refrigerator writes a poem
+refrigerator.writePoem();
+//  action 3: the refrigerator dreams of being a microwave
+refrigerator.dreamOfBeingMicrowave();
+
+//  4)
+//  Noun: television
+//  action 1: the television binge-watches humans
+television.bingeWatchHumans();
+//  action 2: the television changes channels by itself
+television.changeChannelsItself();
+//  action 3: the television pretends to be a radio
+television.pretendToBeRadio();
+
+//  5)
+//  Noun: toaster
+//  action 1: the toaster juggles bread
+toaster.juggleBread();
+//  action 2: the toaster tells a joke
+toaster.tellJoke();
+//  action 3: the toaster takes a selfie
+toaster.takeSelfie();
+
+//  6)
+//  Noun: phone
+//  action 1: the phone calls someone
+phone.call();
+//  action 2: the phone hangs up
+phone.hangUp();   
+//  action 3: the phone dials a number
+phone.dialNumber();
+
+//  7)    
+//  Noun: shoe
+//  action 1: the shoe slides on the floor
+shoe.slideOnFloor();
+//  action 2: the shoe slides on the table
+shoe.slideOnTable();
+//  action 3: the shoe slides on the wall
+shoe.slideOnWall();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.